src/plugins/data_source_management/public/components/direct_query_data_sources_components/direct_query_data_connection/manage_direct_query_data_connections_table.tsx

```diff
@@ -99,28 +145,58 @@ export const ManageDirectQueryDataConnectionsTable: React.FC<ManageDirectQueryDa
       });
   }, [http, notifications.toasts, selectedDataSourceId, featureFlagStatus]);
 
+  const deleteDataSources = useCallback(
+    (connectionName: string | undefined) => {
+      if (!connectionName) return;
+
+      const endpoint =
+        featureFlagStatus && selectedDataSourceId !== undefined
+          ? `${DATACONNECTIONS_BASE}/${connectionName}/dataSourceMDSId=${selectedDataSourceId}`
```

Comment:
I suppose it should be this?
```suggestion
          ? `${DATACONNECTIONS_BASE}/${connectionName}?dataSourceMDSId=${selectedDataSourceId}`
```

Reply:
Reference to this comment: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7419#discussion_r1689406876.

Let's align with what we have for the other endpoints for now. Will add it into followups